### ISSUE TITLE
stm32f3 v0.13.1 seems broken

### ISSUE THREAD

**Issue by Piroro-hs** [stm32-rs/stm32-rs#568]
* f3x4, f301, f302, f303, and f373
Every module is extracted in `src/stm32f3xx/mod.rs` (However, f3x4, f301, and f303 build successfully)
![image](https://user-images.githubusercontent.com/22536104/120894613-44dfc400-c654-11eb-8a95-b91d0c8337f6.png)

* f302 and f373
Module `generic` is declared in `mod.rs` but `generic.rs` does not exist
```rust
#[doc = "Common register and bit access and modify traits"]
pub mod generic;
```

**Comment by adamgreig** [comment]
Sorry, looks like there was an error generating the release package. Hopefully fixed by the just-released 0.13.2.

**Comment by Piroro-hs** [comment]
Fixed! Thanks
